pkg/maps/metricsmap/metricsmap.go

```diff
@@ -40,6 +46,11 @@ const (
 	dirIngress = 1
 	dirEgress  = 2
 	dirUnknown = 0
+
+	// maxCPUs matches the buffer size for cpus.
+	// Reference bpf_num_possible_cpus from
+	// https://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf.git/tree/tools/testing/selftests/bpf/bpf_util.h
+	maxCPUS = 128
```

Comment:
I was initially confused by this because I read the name of the variable and not the comment. A name like `maxCPUs` suggests that it's a maximum number of CPUs supported, but I realised later that it's actually the maximum number of characters supported for one of the `/sys/devices/system/cpu/...` files. Perhaps calling it something like `possibleCPUsFileLength`?

Reply:
sure.

Reply:
done